From: Thomas Renninger <trenn@suse.de>
Subject: Suppress acpi_irq_derive failed message.
Patch-mainline: not yet
References: 154703

Suppress acpi_irq_derive failed message as a warning is printed
out in the above function anyway if no irq could be derived

Signed-off-by: Thomas Renninger <trenn@suse.de>

 drivers/acpi/pci_irq.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

Index: linux-2.6.15/drivers/acpi/pci_irq.c
===================================================================
--- linux-2.6.15.orig/drivers/acpi/pci_irq.c
+++ linux-2.6.15/drivers/acpi/pci_irq.c
@@ -379,8 +379,8 @@ acpi_pci_irq_derive(struct pci_dev *dev,
 	}
 
 	if (irq < 0) {
-		ACPI_WARNING((AE_INFO, "Unable to derive IRQ for device %s",
-			      pci_name(dev)));
+		ACPI_DEBUG_PRINT((ACPI_DB_INFO, "Unable to derive IRQ for device %s",
+				  pci_name(dev)));
 		return_VALUE(-1);
 	}
 
